### PR TITLE
feat(description list): add categorization options

### PR DESCRIPTION
**Pull request by polomani**

### CODE REVIEW

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-455.gearboxjs.preview.cogniteapp.com

**Comment by mrtraser** [comment]
Screen Shot 2019-10-16 at 17 02 45
Can we make it looks prettier?

**Comment by polomani** [comment]
@mrtraser I tried, it keeps messing up indentations, Idk why

File: src/components/DescriptionList/DescriptionList.test.tsx [hunk 1]
```diff
@@ -37,4 +43,78 @@ describe('AssetDetailsPanel', () => {
...
+
+  it("shouldn't render collapse for 0 categories", () => {
+    const wrapper = mount(<DescriptionList valueSet={ASSET_DATA.metadata} />);
+    expect(wrapper.exists(ANT_COLLAPSE_HEADER)).toBeFalsy();
+  });
+
+  it("shouldn't render collapse for 0 categories", () => {
+    const wrapper = mount(<DescriptionList valueSet={ASSET_DATA.metadata} />);
+    expect(wrapper.exists(ANT_COLLAPSE_HEADER)).toBeFalsy();
+  });
```
[the remark this answers is not in the document]
Comment (polomani) [reply src/components/DescriptionList/DescriptionList.test.tsx]: fixed

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-455.gearboxjs.preview.cogniteapp.com

**Comment by polomani** [comment]
@mrtraser fixed indentations as well, we forced to add extra new line apparently

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-455.gearboxjs.preview.cogniteapp.com

File: src/components/AssetDetailsPanel/AssetDetailsPanelPure.tsx [hunk 2]
```diff
@@ -1,15 +1,20 @@
 import React from 'react';
 import { WithAssetDataProps } from '../../hoc/withAsset';
 import { DescriptionList } from '../DescriptionList';
+import { MetaDescriptionListProps } from '../DescriptionList';
+
+export type MetaAssetDetailsProps = MetaDescriptionListProps;
```
Comment (mrtraser) [inline src/components/AssetDetailsPanel/AssetDetailsPanelPure.tsx]: Why do you do this? `MetaDescriptionListProps` also exported

Comment (polomani) [reply src/components/AssetDetailsPanel/AssetDetailsPanelPure.tsx]: will fix

File: src/components/AssetMeta/stories/AssetMeta.stories.tsx [hunk 3]
```diff
@@ -186,7 +206,7 @@ storiesOf('AssetMeta/Examples', module)
     }
   )
   .add(
-    'Custom priority categories',
+    'Docs custom priority categories',
```
Comment (fredrik955) [inline src/components/AssetMeta/stories/AssetMeta.stories.tsx]: Docs custom? Not intuitive what is meant here

File: src/components/AssetMeta/stories/docsCustomCategorySort.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-## Custom Categories Sort
+## Custom Documents Categories Sort
```
Comment (fredrik955) [inline src/components/AssetMeta/stories/docsCustomCategorySort.md]: Not plural

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-455.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-455.gearboxjs.preview.cogniteapp.com

**Comment by cognite-cicd** [comment]
This PR's storybook can now be previewed at the following URL: https://pr-455.gearboxjs.preview.cogniteapp.com

**Pull request merged**